kymatio/scattering1d/frontend/keras_frontend.py

```diff
@@ -36,4 +31,81 @@ def get_config(self):
         keys = ["J", "Q", "max_order", "oversampling"]
         return {key: getattr(self, key) for key in keys}
 
+
+class TimeFrequencyScatteringKeras(ScatteringKeras, TimeFrequencyScatteringBase):
+    def __init__(self, 
+        J,
+        J_fr,
+        Q,
+        #shape,
+        T=None,
+        stride=None,
+        Q_fr=1,
+        F=None,
+        stride_fr=None,
+        #out_type="array",
+        format="time"):
+        #backend="tensorflow",):
+
+        ScatteringKeras.__init__(self)
+        TimeFrequencyScatteringBase.__init__(self, J, J_fr, Q, None, T, stride,
+                Q_fr, F, stride_fr, 'array', format)
+#        self._oversampling = 0
+#        #replace the below code with the above function. 
```

Comment:
remember to build the base class. this will call create_filters and assign fields `phi_f`, `psi1_f`, `psi2_f`, and `filters_fr`

Reply:
should this be built in `__init__` or in `build()`?